src/odbc/odbc.c

```diff
@@ -6709,6 +6805,9 @@ _SQLSetStmtAttr(SQLHSTMT hstmt, SQLINTEGER Attribute, SQLPOINTER ValuePtr, SQLIN
 			break;
 		}
 		break;
+	case SQL_SOPT_SS_PARAM_FOCUS:
+		stmt->param_focus = ui;
+		break;
```

Comment:
I suppose this is also reset resetting the bindings. Need to test. As another comment not a stopover, can be done later